redpen-cli/sample/js/test/validator-js-test.js

```diff
@@ -0,0 +1,52 @@
+/*
+tests validator.js implementation.
+
+steps to run this test:
+ 1. install mocha
+ $ npm install -g mocha
+
+ 2. run RedPen in server mode
+ $ cd $REDPEN_HOME/bin
+ $ ./redpen-server
+
+ 3. rename validator.js.example to enable the validator implementation
+ $ cd $REDPEN_HOME/sample
+ $ mv validator.js.example validator.js
+
+ 4. run mocha
+ $ cd $REDPEN_HOME/sample
+ $ mocha
+
+ */
+assert = require('assert');
```

Comment:
Why is `assert` variable global?

I think that `var assert = require('assert');` is better. 

- http://stackoverflow.com/questions/2485423/is-using-var-to-declare-variables-optional